build.gradle

```diff
@@ -32,6 +33,17 @@ sourceSets {
             include 'images/**'
         }
     }
+    test {
+        java.srcDirs += ['src/test']
```

Comment:
We should also separate test source from regular source as otherwise, test will be part of the "distributed" jar. This would require a bigger refactor though, usually people do something like this:
```
src/
   main/
       java/ 
          org/....
   test/
      java/
         org/
```
Having said that, we could probably go with something simpler and drop the "java" part, i.e:
```
src/ 
    main/
          org/...
    test/
          org/...
   ```